src/main/java/org/intellij/sonar/sonarserver/SonarServer.java

```diff
@@ -39,42 +43,49 @@
     private static final int CONNECT_TIMEOUT_IN_MILLISECONDS = 60*1000;
     private static final int READ_TIMEOUT_IN_MILLISECONDS = 60*1000;
     private final SonarServerConfig mySonarServerConfig;
-    private final Sonar sonar;
-    private final SonarClient sonarClient;
+    private final WsClient sonarClient;
 
     private SonarServer(SonarServerConfig sonarServerConfigBean) {
         this.mySonarServerConfig = sonarServerConfigBean;
-        this.sonar = createSonar();
-        this.sonarClient = createSonarClient(createHost());
+        this.sonarClient = createSonarClient();
     }
 
     public static SonarServer create(SonarServerConfig sonarServerConfigBean) {
         return new SonarServer(sonarServerConfigBean);
     }
 
-    private SonarClient createSonarClient(Host host) {
-        SonarClient.Builder builder = SonarClient.builder()
+    private WsClient createSonarClient() {
```

Comment:
this method is pretty long

Reply:
You think 21 code-lines is too long? o_O

Reply:
looked more then this. up to 30 is ok

Reply:
Even with method declaration and empty lines it is "just" 27 lines. :-)